### DISCUSSION TITLE
Define geometry and create collision

### DISCUSSION THREAD

**Discussion by marioesteban11** [dfki-ric/phobos#288] [Q&A]
Hi everyone, I'm trying to make a simple object in blender to simulate it in GAZEBO. 

I'm using pre_v2.0.0 branch in blender 3.5. I'm trying to set a geometry of the different objets in my model but I dont see/know how to do it. 
![blender_duda_gemotria](https://github.com/dfki-ric/phobos/assets/48151661/42e4927c-8037-4055-91fe-4f15b1ae72a3)

This is my model and I dont really know how to do it.

P.S. I cannot open the Phobos Property information and phobos model warnings tabs, i dont know if its really important or not.

Thanks in advance

**Comment by hwiedPro** [comment]
Have you already read the Phobos Wiki?
There is also a quite good tutorial on Youtube. Have you watched it?
